plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -131,6 +147,8 @@
 
     private final AggregateFunctionRewriter aggregateFunctionRewriter;
 
+    private static final int SQLSERVER_MAX_SUPPORTED_TIMESTAMP_PRECISION = 7;
```

Comment:
nit: Drop the SQLSERVER prefix. Package name gives it away.

Reply:
Done, thanks!